README.md

```diff
@@ -133,6 +133,23 @@ Uncovered API definitions found:
 /v2/pet/:petId/uploadImage | POST       | 200         
 ```
 
+# Exporting the report: 
+When providing `reportCoverage:true`, `exportCoverage: true` a `coverage.json` file will be created in your cwd with following structure: 
```

Comment:
description is a little confusing as it sounds that both are required in order to export the report

Reply:
done